src/helper/remote_config/protocol/cached_target_files.hpp

```diff
@@ -0,0 +1,30 @@
+// Unless explicitly stated otherwise all files in this repository are
+// dual-licensed under the Apache-2.0 License or BSD-3-Clause License.
+//
+// This product includes software developed at Datadog
+// (https://www.datadoghq.com/). Copyright 2021 Datadog, Inc.
+#pragma once
+
+#include <list>
+#include <string>
+
+#include "cached_target_file_hash.hpp"
+
+namespace dds::remote_config::protocol {
+
+class cached_target_files {
+public:
+    cached_target_files(std::string path, int length,
```

Comment:
Try not to name parameters the same as members of the class as it gets confusing and could potentially result in unexpected outcomes.

Reply:
Done